pymaker/proxy.py

```diff
@@ -0,0 +1,236 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2018 bargst
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import logging
+import re
+from typing import List
+
+from hexbytes import HexBytes
+from web3 import Web3
+from web3.utils.contracts import get_function_info, encode_abi
+from web3.utils.events import get_event_data
+
+from pymaker import Address, Contract, Transact, Receipt
+
+
+def code_to_bytes(code: str):
```

Comment:
How is this different from `hexstring_to_bytes` which we already have in `utils.py`... ?

Reply:
The diff is that it allow code to not start with '0x' and use the `Web3.toBytes()` function. I see 2 option:
* Update test code to add '0x' and use the `hexstring_to_bytes` as is
* Update hexstring_to_bytes to use the same code_to_bytes way of conversion

Any pref ?